packages/fether-react/src/Accounts/AccountsList/AccountsList.js

```diff
@@ -9,6 +9,7 @@ import { accountsInfo$, withoutLoading } from '@parity/light.js';
 import { inject, observer } from 'mobx-react';
 import light from '@parity/light.js-react';
 
+import Feedback from '../../Feedback';
```

Comment:
Could you put Feedback in `fether-react/src/Accounts/AccountsList/Feedback/Feedback.js`, since it's only used in `AccountsList` component?